packages/ui/components/icon/Discord.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { forwardRef } from "react";
+
+export const Discord = forwardRef<SVGSVGElement, React.SVGProps<SVGSVGElement>>(function Discord(props, ref) {
+  return (
```

Comment:
It could be a static asset in public/discord.svg which is cacheable then. We can then use it as img src and add classes to it.

Reply:
i like this better

Reply:
lets add this to public/discord.svg

Reply:
hey i will make a follow up PR making these changes